### ISSUE TITLE
Dashboard: lack of color contrast for status filter

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#8669]
## Bug Description

One can barely see the number of stories when a status is selected. It's dark gray on black.

## Screenshots

![Screenshot 2021-08-11 at 15 50 10](https://user-images.githubusercontent.com/841956/129210766-9ad847ce-54eb-4f8d-bdf6-bf3302a7aa3e.png)

## Additional Context

- Plugin Version: 1.10.0
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by BrittanyIRL** [comment]
@aaskedall could you help us with this one? This is a color the former design team told us to use, despite the lack of contrast 😞  I feel like we could swap it out.

This is gray 20 (#ADB1B3) as the number and gray 90 (#131516) background, #fff is the text color.
<img width="160" alt="Screen Shot 2021-08-23 at 8 49 46 AM" src="https://user-images.githubusercontent.com/10720454/130477730-0c24c119-729c-41c9-9b28-bfb3290d2b21.png">
https://webaim.org/resources/contrastchecker/?fcolor=ADB1B3&bcolor=131516


And this is gray 60, #fff background. The text is gray 70 to the left. 
<img width="97" alt="Screen Shot 2021-08-23 at 8 53 15 AM" src="https://user-images.githubusercontent.com/10720454/130478297-a07332f3-e334-4c56-8eb7-70243ae98c1a.png">
https://webaim.org/resources/contrastchecker/?fcolor=4B5253&bcolor=FFFFFF


These are from the gray palette in the figma design system, based on the light theme.  This component is a variation of the `Pill` in the design system.

The color difference is so slight with this suggestion, I am tempted to just have the number be the same color as the text as it's part of the same display. Would love your expertise here. thanks!

**Comment by aaskedall** [comment]
Chiming in. 

![image](https://user-images.githubusercontent.com/4030811/130481730-d3660e93-821e-4368-8479-83057de95bad.png)
 Seems to be a color in our style guide with enough contrast. 
#C8CBCC

![image](https://user-images.githubusercontent.com/4030811/130481696-48cd4a27-5043-4c05-859b-794d43aef092.png)

**Comment by BrittanyIRL** [comment]
Thanks! yeah gray 10 and 20 both fit the contrast needs for the selected button (see my comment above).

Gray 10 doesn't work with the unselected pill though. https://webaim.org/resources/contrastchecker/?fcolor=C8CBCC&bcolor=FFFFFF

**Comment by BrittanyIRL** [comment]
Wanted to circle back to this, it should be an easy one to get fixed. @agingoldseco - when you have a sec!

**Comment by BrittanyIRL** [comment]
i think pea pod can just take a stab at this and send screenshots to confirm updates.

**Comment by BrittanyIRL** [comment]
Hey @agingoldseco @aaskedall  - does this work? 

| Current | Proposed Change | 
| --- | --- | 
| <img width="374" alt="Screen Shot 2021-12-06 at 4 06 51 PM" src="https://user-images.githubusercontent.com/10720454/144937211-c0c4501e-f642-4f85-a6e9-e59139fcbbab.png"> | <img width="374" alt="Screen Shot 2021-12-06 at 4 06 26 PM" src="https://user-images.githubusercontent.com/10720454/144937226-0bfee47b-aead-49c9-9fee-4fab91602910.png"> |

Swapped the active view's count to be `gray.20` (#ADB1B3) so the contrast ratio is 8.47 vs 3.11 with `gray.50` (foreground tertiary #5E6668). gray.50 is great for the inactive counts.

**Comment by agingoldseco** [comment]
@BrittanyIRL Yes, imo that absolutely works!
